source/tests/test.ts

```diff
@@ -454,6 +455,14 @@ if (isNode8orHigher) {
 	});*/
 }
 
+if (isNode10orHigher) {
+	test('is.asyncIterable', t => {
+		t.true(m.asyncIterable(Object.create({
```

Comment:
I don't see the point of using `Object.create()` here. It could just be a plain object literal.

Reply:
@sindresorhus, Yep, you're right